recipe/meta.yaml

```diff
@@ -93,9 +93,12 @@ outputs:
         - python
       run:
         - python
-        - pyqt >=5.10  # [not osx and not aarch64 and not ppc64le and python_impl == "cpython"]
+        # osx has its own backend; pyside6 not available on ppc
+        - pyside6 >=6.7.2  # [python_impl == "cpython" and not (osx or ppc64le)]
         - tornado >=5
         - {{ pin_subpackage('matplotlib-base', max_pin="x.x.x") }}
+      run_constrained:
+        - pyside6 != 6.7.2=*_0
```

Comment:
Looks like it did not like this:
```
The following package could not be installed
└─ matplotlib 3.9.1.* py310hff52083_2 is installable and it requires
   └─ pyside6 [!=6.7.2 *_0|>=6.7.2 ] with the potential options
      ├─ pyside6 6.7.2, which can be installed;
      └─ pyside6 !=6.7.2 conflicts with any installable versions previously reported.
```
Looks like the `!=` ignores the build number specification?